CavalcanteLucas/todolist
# Update/OL93-boilerplate_fixes

**Link to task:** 
https://labcodes.atlassian.net/browse/OL-93

**How to test:**
- Check if there are no `blog_post` files anymore;
- Check if Luciano is now listed as an author in `README.md`;
- Check if original boilerplate link is mentioned in `README.md`
- Run `create_project.sh` script and check that the project is running at `localhost:8000`

**Description of your solution:**
Noticed that blogposts from original boilerplate were included in this project, so removed those. The authors list has been updated to include Luciano. Also, the original boilerplate is now mentioned in the `README.md`. Finally, with this PR one can now run the `create_project.sh` script to have this project running locally.
